src/assets/scss/components/search.scss

```diff
@@ -0,0 +1,71 @@
+.search-input {
+  border: 0;
+  border-bottom: 2px solid $dark-color;
+  border-radius: 0;
+  justify-content: space-between;
+  outline: none;
+  padding: 0px 10px;
+  padding-left: 0px;
+  position: relative;
+  width: 290px;
+}
+
+.search-input .filter-title {
```

Comment:
we are using scss, we should try to use nesting here. As the app grows, this will be hard to maintain